docs/UserGuide.md

```diff
@@ -228,8 +240,8 @@ Deletes an order from the list of tracked orders.
 Format: `deleteo INDEX`
 
 * Deletes the order at the specified INDEX.
```

Comment:
I think it would be good to make the markdown consistent here

`INDEX` instead of INDEX

Do help to check all instances of this